src/app/[locale]/specs/account/page.mdx

```diff
@@ -0,0 +1,90 @@
+
+By default, if an account’s current active PDS is not available, account status should remain unchanged, at least for some time period. This keeps the network resilient and accounts online during temporary infrastructure outages. Services may decide their own policies for accounts whose PDS hosts remain offline for long periods of time.
+
+The general expectation is that downstream services will not list an account as `active` if it is inactive at an upstream service. However, the concept of “upstream” is informal, and services may need to define their own policies for some corner-cases. For example, an AppView might consume from multiple independent relay providers, who report differing statuses for the same account, due to policy differences. If the network connection between a relay and PDS is disruptive, they may report different statuses for the same account. 
+
+If there is doubt about about an account’s general status in the network, the account’s current active PDS host can be queried using the `com.atproto.sync.getRepoStatus` endpoint. If the account is not `active` there, it is generally expected to not be active elsewhere, especially in the case of user-initiated actions (deactivation or deletion).
+
+### Content Deletion
+
+Separate from overall account status, individual pieces of content may be removed. When accounts remove records from their public repositories, hosting services should remove public access to that content.
+
+Applications which would benefit from explicit “tombstones” (indications that content previously existed) should explicitly design them in to Lexicon schemas. In the absence of explicit application-level tombstones, services are expected not to differentiate between content which has never existed and content which has been entirely deleted. Note that this is different from acknowledging account-level deletion, which is encouraged.
+
+## PDS Account Migration
+
+At a high level, the current PDS host for an account is indicated in the account’s DID document. If a DID document is updated to point at a new PDS host, the account status is `active` at that host, and there is a functioning repository (with valid signature and commit `rev`), then the account has effectively been migrated.
+
+At a protocol level, this is how account migration works. And in some situations, when any previous PDS instance is unavailable or has inactive account status, getting in to the above state may be the only account recovery path available.
+
+However, in the common case, when there is an active account on a functional current PDS, a more seamless account migration process is possible. It can summarized in a few high-level steps:
+
+- creating an account on the new PDS (which may start with `active` false, such as `deactivated`)
+- migrating data from the old PDS to the new PDS
```

Comment:
It's been interesting to see the deprioritization of storing a full copy of your repo on your local devices to support migration. Totally ok! Just interesting. https://atproto.com/guides/overview#:~:text=backup

Reply:
if both PDS instances are online, then we'd probably always recommend PDS-to-PDS migration, it will just be faster and use less bandwidth/battery on a phone (for example).

we do want to have repos backed up more places under user control! in a many-app world, i'm not sure having individual apps handle this makes the most sense though. today it probably does make sense for bsky app to do this, because it is so popular. but should whtwnd, smokesignals, etc, all do full local on-device CAR file backups? maybe a dedicated mobile app for "atproto account control and backup" makes more sense. could also PLC identity key management, and maybe even E2EE DMs in the future; the later two are security sensitive and might benefit from being outside something like the Bluesky Social app (which has large feature surface).